packages/sdds-insol-next/scripts/copy-linaria-components.sh

```
@@ -1,15 +1,15 @@
 #!/usr/bin/env bash
 
 # Example: Calendar
-components=$(grep -R plasma-new-hope src/components | cut -d / -f 3 | sort -u)
+components=$(grep -R plasma-new-hope src/components | cut -d / -f 3 | grep -v '^_beta$' | sort -u)
 
 mkdir -p src-css/components/
 touch src-css/index.ts
 touch src-css/index.d.ts
 for component in $components; do
         cp -R src/components/$component src-css/components/;
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)